org.gnome.timetrack.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "org.gnome.timetrack",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gnome-timetrack",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=xdg-run/gvfs",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--talk-name=ca.desrt.dconf",
+        "--talk-name=org.gtk.vfs",
```

Comment:
are the gtk.vfs really needed?

Reply:
I don't think so, the app only need access to the filesystem to write the database file here:

```
os.path.join(GLib.get_user_data_dir(), "timetrack.sqlite3")
```